File: components/credentials/src/main/java/org/cloudfoundry/credhub/audit/entities/GetAllCertificates.java
package org.cloudfoundry.credhub.audit.entities;

import org.cloudfoundry.credhub.audit.OperationDeviceAction;
import org.cloudfoundry.credhub.audit.RequestDetails;

public class GetAllCertificates implements RequestDetails {

  @Override
  public OperationDeviceAction operation() {
    return OperationDeviceAction.GET_ALL_CERTIFICATES;
  }
}

File: components/credentials/src/main/kotlin/org/cloudfoundry/credhub/audit/entities/GetAllCertificates.kt
package org.cloudfoundry.credhub.audit.entities

import org.cloudfoundry.credhub.audit.OperationDeviceAction
import org.cloudfoundry.credhub.audit.RequestDetails

class GetAllCertificates : RequestDetails {
    override fun operation(): OperationDeviceAction = OperationDeviceAction.GET_ALL_CERTIFICATES
}
